### DISCUSSION TITLE
GPUDirect RDMA

### DISCUSSION THREAD

**Discussion by hamedhighpc** [NVIDIA/open-gpu-kernel-modules#699] [Q&A] [answered]
By enabling the P2P feature, will feature GPUDirect RDMA also be activated for the RTX 4090 graphics card? 
The NVIDIA documentation states that GPUDirect RDMA is applicable to the Tesla and Quadro families. However, this feature also depends on DMA-BUF and P2P.

**Comment by bmaxdk** [comment, marked as answer]
Enabling P2P alone won't get you GPUDirect RDMA on the RTX 4090. That feature is mainly for Tesla and Quadro GPUs, as they have the required hardware support. The RTX 4090 indeed powrful but it is designed more for gaming and creative workloads, not for the HPC scenarios where GPUDirect RDMA is typically used. So, even with P2P turned on, don't expect RDMA to work on an RTX 4090.
